textattack/constraints/semantics/sentence_encoders/multilingual_universal_sentence_encoder/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .multilingual_universal_sentence_encoder import (
```

Comment:
I'm not sure if it's necessary to create new a directory for MUSE. Since MUSE is an extension of USE and since we already have a directory for USE, it seems to make more sense to just have both MUSE and USE under `textattack/constraints/semantics/sentence_encoders/universal_sentence_encoder`.

Reply:
I'm fine with this as well, how would you distinguish between them when calling on the command line?
As another argument?

Reply:
Ah, got your point. Sure i can move it to the other directory. :)

Reply:
Fixed it.